apps/app/src/app/pages/(components)/components/(date-picker)/date-picker.page.ts

```diff
@@ -0,0 +1,135 @@
+import type { RouteMeta } from '@analogjs/router';
+import { Component } from '@angular/core';
+import { hlmCode, hlmH4, hlmP, hlmUl } from '@spartan-ng/ui-typography-helm';
+import { CodeComponent } from '../../../../shared/code/code.component';
+import { PageBottomNavLinkComponent } from '../../../../shared/layout/page-bottom-nav/page-bottom-nav-link.component';
+import { PageBottomNavComponent } from '../../../../shared/layout/page-bottom-nav/page-bottom-nav.component';
+import { PageNavComponent } from '../../../../shared/layout/page-nav/page-nav.component';
+import { SectionIntroComponent } from '../../../../shared/layout/section-intro.component';
+import { SectionSubHeadingComponent } from '../../../../shared/layout/section-sub-heading.component';
+import { TabsCliComponent } from '../../../../shared/layout/tabs-cli.component';
+import { TabsComponent } from '../../../../shared/layout/tabs.component';
+import { metaWith } from '../../../../shared/meta/meta.util';
+import { datePickerConfigCode, DatePickerConfigExampleComponent } from './date-picker--config.example';
+import { datePickerFormCode, DatePickerFormExampleComponent } from './date-picker--form.example';
+import { datePickerFormatCode, DatePickerFormatExampleComponent } from './date-picker--format.example';
+import { codeSkeleton, DatePickerPreviewComponent, defaultCode, defaultImports } from './date-picker.preview';
+
+export const routeMeta: RouteMeta = {
+	data: { breadcrumb: 'Date Picker' },
+	meta: metaWith('spartan/ui - Date Picker', 'A date picker component.'),
+	title: 'spartan/ui - Date Picker',
+};
+
+@Component({
+	selector: 'spartan-calendar',
+	standalone: true,
+	imports: [
+		DatePickerPreviewComponent,
+		SectionIntroComponent,
+		TabsComponent,
+		CodeComponent,
+		SectionSubHeadingComponent,
+		TabsCliComponent,
+		PageBottomNavComponent,
+		PageBottomNavLinkComponent,
+		PageNavComponent,
+		DatePickerConfigExampleComponent,
+		DatePickerFormatExampleComponent,
+		DatePickerFormExampleComponent,
+	],
+	template: `
+		<section spartanMainSection>
+			<spartan-section-intro name="Date Picker" lead="A date picker component." />
+
+			<spartan-tabs firstTab="Preview" secondTab="Code">
+				<div spartanCodePreview firstTab>
+					<spartan-date-picker-preview />
+				</div>
+				<spartan-code secondTab [code]="defaultCode" />
+			</spartan-tabs>
+
+			<spartan-section-sub-heading id="installation">Installation</spartan-section-sub-heading>
+			<spartan-cli-tabs
+				class="mt-4"
+				nxCode="npx nx g @spartan-ng/cli:ui date-picker"
+				ngCode="ng g @spartan-ng/cli:ui date-picker"
```

Comment:
I don't think the CLI ui generator has been updated to support this component.

Reply:
Yes totally forgot about that! I added it to the generator and updated the primitive dependencies as well